contracts/modules/ProtocolSettings.sol

```diff
@@ -863,4 +866,75 @@ contract ProtocolSettings is
             newRolloverFlexFeePercent
         );
     }
+
+    /**
+     * @dev Get default path conversion for pairs.
+     *
+     * @param sourceTokenAddress source token address.
+     * @param destTokenAddress destination token address.
+     *
+     * @return default path of the conversion.
+     */
+    function getDefaultPathConversion(address sourceTokenAddress, address destTokenAddress)
+        external
+        view
+        returns (IERC20[] memory)
+    {
+        return defaultPathConversion[sourceTokenAddress][destTokenAddress];
+    }
+
+    /**
+     * @dev Set default path conversion for pairs.
+     *
+     * @param sourceTokenAddress source token address.
+     * @param destTokenAddress destination token address.
+     * @param defaultPath array of addresses for the default path.
+     *
+     */
+    function setDefaultPathConversion(
+        address sourceTokenAddress,
+        address destTokenAddress,
+        IERC20[] calldata defaultPath
+    ) external onlyOwner whenNotPaused {
+        require(
+            Address.isContract(sourceTokenAddress) && Address.isContract(destTokenAddress),
+            "ERR_NON_CONTRACT_ADDR"
+        );
+
+        uint256 defaultPathLength = defaultPath.length;
+        require(defaultPathLength >= 3, "ERR_PATH_LENGTH");
```

Comment:
```suggestion
        require(defaultPathLength > 0, "ERR_PATH_LENGTH");
```
the first and the last elements in the defaultPath are always source and dest tokens
consider passing only the path between them to remove duplication when passing them also in the defaultPath param.

Reply:
Yes, the source & dest tokens would in the first & last index, but in the case of the minimum path length, it will always be 3 at minimum.

Reply:
> pls consider passing only the path between them to remove duplication when passing them also in the defaultPath param.

Reply:
Need to remove the source & dest token address params, use the first & n length array instead